apps/docsite/package.json

```diff
@@ -2,5 +2,32 @@
   "name": "@alma-oss/spirit-docsite",
   "version": "0.0.0",
   "private": true,
-  "type": "module"
+  "type": "module",
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build",
+    "start": "next start",
+    "lint": "ESLINT_USE_FLAT_CONFIG=false next lint"
+  },
+  "dependencies": {
+    "@lmc-eu/spirit-design-tokens": "workspace:^",
+    "@lmc-eu/spirit-icons": "workspace:^",
+    "@lmc-eu/spirit-web": "workspace:^",
+    "@lmc-eu/spirit-web-react": "workspace:^",
+    "next": "15.5.6",
+    "react": "19.2.0",
+    "react-dom": "19.2.0",
+    "sass-embedded": "1.93.2"
+  },
+  "devDependencies": {
+    "@next/eslint-plugin-next": "14.2.33",
+    "@types/node": "20.17.24",
+    "@types/react": "18.3.18",
+    "@types/react-dom": "18.3.5",
+    "clsx": "2.1.1",
+    "eslint": "8.57.1",
```

Comment:
🤞🏻 https://github.com/lmc-eu/spirit-design-system/pull/2122

Reply:
Refactored to flat config. Ready for ESLint v9.